translator/internaldata/oc_to_traces.go

```diff
@@ -231,21 +231,17 @@ func initAttributeMapFromOC(ocAttrs *octrace.Span_Attributes, dest pdata.Attribu
 	}
 
 	if len(ocAttrs.AttributeMap) > 0 {
-		dest.InitEmptyWithCapacity(len(ocAttrs.AttributeMap))
+		dest.EnsureCapacity(len(ocAttrs.AttributeMap))
```

Comment:
Same.

Reply:
Not a problem, the output param is empty.